spec/index.html

```diff
@@ -3405,8 +3405,12 @@ <h2>Serializing an RDF Graph to RDF/XML</h2>
       <a href="#syntaxTerms">syntaxTerms</a> production.
       </dd>
 
-      <dt>Use the <code>rdf:HTML</code> datatype</dt>
-      <dd>This datatype as introduced in RDF 1.2 [[RDF12-CONCEPTS]].</dd>
+      <dt>Use the <code>rdf:HTML</code> datatype along with <code>rdf:parseType="Literal"</code></dt>
+      <dd>This datatype as introduced in RDF 1.1 [[RDF11-CONCEPTS]] for representing
+        HTML <a data-cite="DOM#interface-documentfragment"><code>DocumentFragment</code></a>
+        nodes as literal values using HTML markup.
+        Literals with datatype `rdf:HTML` can be represented as normal
+        <a href="#section-Syntax-datatyped-literals">typed literals</a>.</dd>
```

Comment:
This paragraph is a bit weird to me because it is only stating that a particular serialization mechanism is not possible, not that some RDF triples are not serializable whereas the list is introduced by <q>...that cannot be serialized in RDF/XML. These are those that:</q> . The paragraph would make as much sense stating that "all datatypes but `rdf:XMLLiteral` cannot be serialized with `rdf:parseType="Literal"`. I would tend to think we should just remove the paragraph entirely.

Reply:
I'm going with @afs's suggestion to just drop the note on `rdf:HTML` entirely.